### PR TITLE
Add executable `mule` that runs each pack

### PR DESCRIPTION
**Pull request by jwaiton**
I've added the skeleton for the executable `mule`. This command asks for:
- The pack you want to use
- The config file you want to use

For example, in the acquisition pack `acq/`, it will execute script `acq` within `acq.py`, which **should** read the config and run whatever scripts accordingly. This can be tested at the moment.

This enforces a strict setup for what a pack should do (collect a config and run an application upon it), which I think will be useful for structuring the repository.

### CODE REVIEW

File: bin/mule [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+======================================
+MULE pack runtime executable
+Use 'mule --help' for more information
+======================================''', formatter_class=argparse.RawTextHelpFormatter)
+
+parser.add_argument("pack", choices = ['acq','proc'], help = '''The pack implemented: 
+        acq  - Acquisition of data using wavedump 1
+        proc - Processing of data
+        ''')
+parser.add_argument("config", help = 'The config file provided to the pack, this differs based on which pack youre using')
```
Comment (bpalmeiro) [inline bin/mule:34]: maybe add it as "-p" "--pack" instead so it's more visual? same for config. Not a strong opinion tho

**Review by bpalmeiro — CHANGES_REQUESTED** [review]
you can test the two arguments and the failure I guess, other than that, good to go

**Comment by jwaiton** [comment]
Failure of the config parameter will be tested later, and has been added as an issue ([#9](https://github.com/nu-ZOO/MULE/issues/9)).

**Review by bpalmeiro — APPROVED** [review]
Very goodio

**Pull request merged**